src/PrestaShopBundle/Resources/views/Admin/AdvancedParameters/system_information.html.twig

```diff
@@ -0,0 +1,287 @@
+                                    <li>{{ requirements.testsErrors[key] }}</li>
+                                {% endif %}
+                            {% endfor %}
+                        </ul>
+                    {% endif %}
+                    {% if requirements.failOptional is defined %}
+                        <p>
+                        <strong>{{ 'Optional parameters:'|trans }}</strong>
+                        {% if requirements.failOptional == false %}
+                            <span class="text-success">{{ 'Ok'|trans({}, 'Admin.Advparameters.Notification') }}</span>
+                            </p>
+                        {% else %}
+                            <span class="text-danger">{{ 'Please fix the following error(s)'|trans({}, 'Admin.Advparameters.Notification') }}</span>
+                            </p>
+                            <ul>
+                                {% for key, value in requirements.testsOptional %}
+                                    {% if 'fail' == value and testsErrors[key] is defined %}
+                                        <li>{{ requirements.testsErrors[key] }}</li>
+                                    {% endif %}
+                                {% endfor %}
+                            </ul>
+                        {% endif %}
+                    {% endif %}
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+{% if system.notHostMode %}
+    <div class="card">
+        <h3 class="card-header">
+            <i class="material-icons">info_outline</i> {{ 'List of changed files'|trans }}
+        </h3>
+        <div class="card-block">
+            <div class="card-text" id="changedFiles">
+                <i class="material-icons">loop</i> {{ 'Checking files...'|trans({}, 'Admin.Advparameters.Notification') }}
+            </div>
+        </div>
+    </div>
+{% endif %}
+{% if system.notHostMode %}
+    <script>
+        $(document).ready(function()
+        {
+            var translations = {
+                missing: '{{ "Missing files"|trans({}, "Admin.Advparameters.Notification")|e('js') }}',
+                updated: '{{ "Updated files"|trans({}, "Admin.Advparameters.Notification")|e('js') }}',
+                changesDetected: '{{ "Changed/missing files have been detected."|trans({}, "Admin.Advparameters.Notification")|e('js') }}',
+                noChangeDetected: '{{ "No change has been detected in your files."|trans({}, "Admin.Advparameters.Notification")|e('js') }}'
+            };
+
+            $.ajax({
+                type: 'POST',
+                url: '{{ path("admin_system_information_check_files") }}',
+                data: {},
+                dataType: 'json',
+                success: function(json)
+                {
+                    var tab = {
+                        'missing': translations.missing,
+                        'updated': translations.updated
```

Comment:
Missing comma after the last item according to [airbnb rules](https://github.com/airbnb/javascript#commas--dangling)